allure-commandline/src/main/java/ru/yandex/qatools/allure/command/AllureCommandException.java

```diff
@@ -0,0 +1,20 @@
+package ru.yandex.qatools.allure.command;
+
+/**
+ * eroshenkoam
+ * 13.08.15
```

Comment:
there is no reason to use this comment - the date is lie and `eroshenkoam` says nothing for all the world